### PR TITLE
obs_lsstSim/DM-7884: unification of coadd and diff datasets

### PR DESCRIPTION
**Pull request by TallJimbo**

### CODE REVIEW

File: policy/LsstSimMapper.paf [hunk 1]
```diff
@@ -125,11 +117,8 @@ exposures: {
         level:       "None"
         tables:      raw
     }
+    # dataset defined in obs_base modified in this mapper as follows:
```
Comment (TallJimbo) [inline policy/LsstSimMapper.paf]: As in `obs_decam`, there's no reason to specialize `level`.

File: policy/LsstSimMapper.paf [hunk 2]
```diff
@@ -672,378 +532,46 @@ datasets: {
...
     deepCoadd_forced_src: {
         template:      "deepCoadd-results/%(filter)s/%(tract)d/%(patch)s/forced-%(filter)s-%(tract)d-%(patch)s.fits"
-        python:        "lsst.afw.table.SourceCatalog"
-        persistable:        "ignored"
-        storage:    "FitsCatalogStorage"
     }
+    # dataset defined in obs_base modified in this mapper as follows:
     deepCoadd_forced_config: {
         template:      "config/deepCoadd_forced.py"
         python:        "lsst.meas.base.forcedCoadd.ForcedCoaddMeasurementConfig"
```
Comment (TallJimbo) [inline policy/LsstSimMapper.paf]: I think the `python` entry here must either be incorrect or the `obs_base` one is.  Probably the former.

Comment (pgee2000) [inline policy/LsstSimMapper.paf]: Already noted in RFC-237

File: policy/LsstSimMapper.paf (hunk 1 above)
Comment (pgee2000) [inline policy/LsstSimMapper.paf]: Added to RFC-237

**Pull request merged**